tests/graph/tests/integration/graph/edge-removal/abstract-edge-removal-test.ts

```diff
@@ -116,7 +116,7 @@ module('Integration | Graph | Edge Removal', function (hooks) {
          * However: for a newly created record any form of rollback, unload or persisted delete
          * will result in it being destroyed and cleared
          */
-        await testFinalState(
+        testFinalState(
```

Comment:
IDK why the linter was complaining about these in this PR, but this change appears correct based on the code.